pom.xml

```diff
@@ -97,7 +97,7 @@
         <global_hadoop_version>${base_hadoop_version}</global_hadoop_version>
         <global_hbase_version>${base_hbase_version}</global_hbase_version>
         <global_flume_version>${base_flume_version}</global_flume_version>
-        <global_elasticsearch_version>5.6.2</global_elasticsearch_version>
+        <global_elasticsearch_version>5.6.7</global_elasticsearch_version>
```

Comment:
Just noticed this - what's the motivation for changing ES versions as part of this PR?

Reply:
There was no x-pack transport client build for 5.6.2